### ISSUE TITLE
[BUG] 模板列表页"芯片综合评测"显示19评测项，创建向导显示21评测项（#468 回归）

### ISSUE THREAD

**Issue by chenxibj** [chenxibj/ai-hardware-verification-platform#474]
## 问题描述

第1轮测试发现的 #468 未完全修复。模板列表页（TemplateList.js）和创建向导（PlanCreate.js）对"芯片综合评测"模板的评测项计数不一致。

- 模板列表页显示：**19 评测项**
- 创建向导显示：**21 评测项**

## 根因分析

TemplateList.js 第292行的计算逻辑缺少 `training` 项：
```js
// TemplateList.js line 292 - 缺少 training
const itemCount = config.itemCount || (config.operators?.length || 0) + (config.models?.length || 0) + hfModels.length;
```

PlanCreate.js 第533行正确包含了 `training`：
```js
// PlanCreate.js line 533 - 正确
const itemCount = config.itemCount || (config.operators?.length || 0) + (config.models?.length || 0) + (config.training?.length || 0);
```

"芯片综合评测"模板的 configJson 包含：
- operators: 16个 (MatMul, Conv2D, Softmax, ReLU, GELU, SiLU, LayerNorm, BatchNorm, Attention, ScaledDotProduct, Add, Mul, Transpose, Gather, Embedding, Linear)
- models: 3个 (MLP-Small, ResNet-50, BERT-Base)
- training: 2个 (MLP-Small-Train, ResNet-50-Finetune)
- 正确总数 = 16 + 3 + 2 = **21**

## 修复建议

TemplateList.js 第292行改为：
```js
const itemCount = config.itemCount || (config.operators?.length || 0) + (config.models?.length || 0) + (config.training?.length || 0) + hfModels.length;
```

## 复现步骤

1. 打开模板管理页 `/templates`
2. 查看"芯片综合评测"模板卡片，显示 "19 评测项"
3. 打开创建任务页 `/plans/create`，选择芯片后进入模板选择
4. 查看"芯片综合评测"模板卡片，显示 "21 评测项"

## 预期行为

两处显示应一致，均为 21 评测项

## 实际行为

模板列表 19，创建向导 21

## 环境信息

- 平台版本: 2b3ff88e
- 测试时间: 2026-04-17
- 关联 issue: #468
